### PR TITLE
Added CLI to hw_sd_list, mainly for filtering by session name.

### PR DESCRIPTION
**Pull request by hernando**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 # git master
 
+* [49](https://github.com/Eyescale/hwsd/pull/47): Added a command line options
+  to hw_sd_list. Now results can be filetered by session name.
```
Comment (tribal-tec) [inline doc/Changelog.md]: typo 'filetered'

**Comment by tribal-tec** [comment]
Otherwise fine with me. Do we need a version bump?

**Comment by hernando** [comment]
I don't know. For libraries it's clear when to bump each version number, but not for applications.

**Comment by eile** [comment]
+1 otherwise.

**Comment by eile** [comment]
Re version: same logic. Bump minor when you have a new feature, major if it's incompatible (ie the user experience changed drastically)

**Comment by eile** [comment]
I guess we'll merge after release?

**Comment by hernando** [comment]
I this case it's a bit funny because it's a new feature, but only in one of the tools, and bumping the minor version number without bumping the ABI version looks odd. Should apps have their own subproject (but still inside the same source tree)?

**Comment by hernando** [comment]
I bumped the minor version, ready to merge?

**Pull request merged**